### PR TITLE
New build system

### PR DESCRIPTION
**Pull request by shigma**
- Use mathematica to generate symbol lists and inserted them into syntax file automatically. Currently support variables `built_in_functions`, `built_in_options`, `built_in_constants` and `named_characters`. A `.mx` file will be generated to collect data for all this, which is git-ignored and can be regenerated by running `namespace.wl`.
- Change the language name into `Wolfram Language`. (Is that necessary?)
- Add suffix `.wolfram` to some of the scopes.
- Bump Mathematica version into 11.3.

### CODE REVIEW

**Comment by chere005** [comment]
How do you get the usages from sublime? Is the syntax settings etc on this branch already built, or does it need to be built in order to test? How are the usages pulled up in sublime? Is there documentation for the build system committed yet that I just missed?

File: build/namespace.wl [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
-]];
+namedCharacters = StringTake[getAtomic[#, {1, 1, -1, 1}], {3, -2}]& /@ Import[FileNameJoin[{
+	$InstallationDirectory,
+	"Documentation",
+	$Language,
+	"System/Guides/ListingOfNamedCharacters.nb"
+}], {"Cells", "GuideText"}];
+
+
+DumpSave[NotebookDirectory[] <> "wldata.mx", "wl`"];
```
Comment (chere005) [inline build/namespace.wl]: Should there be a notebook committed if this file expects one to be saved?

**Comment by shigma** [comment]
> How do you get the usages from sublime? Is the syntax settings etc on this branch already built, or does it need to be built in order to test? How are the usages pulled up in sublime? Is there documentation for the build system committed yet that I just missed?

I haven't done all the jobs for the usages before they can be seen in sublime (current jobs can be seen at *plugin-initiate* branch), so the usage file is currently useless. If you mind, I can remove it now and wait for the plugin coming up.

File: build/namespace.wl (hunk 1 above)
Comment (shigma) [reply build/namespace.wl]: No, the *wldata.mx* will only be used by *classify.wl*, not by sublime. Anyone who wants the file can generate it by himself. Given that the file is big and binary, a better practice is to leave it ignored.

**Comment by chere005** [comment]
I might take a crack at a build system myself, this seems a bit hard to follow and lacks proper documentation of the build process. Also need to update for 12 now.
